docs/app/pages/components/components-sections/dashboard/dashboard/snippets/aria.html

```diff
@@ -0,0 +1,12 @@
+...
+
+<button class="widget-grab-handle"
+    uxDashboardGrabHandle
+    uxDashboardWidgetDragHandle
+    uxGrabAriaLabel="'Drag Handle'"
+    uxGrabChangeSuccessAnnouncement="'The widget has been sucessfully moved or resized'"
```

Comment:
These inputs accept a function so that the string can be dynamic and give the user accurate information. An example of that would be more helpful than generic strings.